spa-react/src/hooks/use-setUsers.hook.ts

```diff
@@ -0,0 +1,21 @@
+import { doc, setDoc } from "firebase/firestore";
+import { IUser } from "src/components/TableWithUsers/types";
+import { db } from "src/utils/firebase";
+
+interface IUsers {
+  users: IUser[];
+}
+
+export const useSetUsersToDb = (users: IUsers) => {
+  const setUsersToDB = async () => {
+    try {
+      const docRef = await setDoc(doc(db, "usersDB", "users"), users);
```

Comment:
Each user should be written as new doc with unique id

Reply:
if there is no such doc, firestore will create it. I checked it, so we dont need to write condition for checking the existence of the doc